app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/upload/UploadSuccessQrScreen.kt

```diff
@@ -41,8 +45,17 @@ import com.infomaniak.swisstransfer.ui.theme.Margin
 import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
 import com.infomaniak.swisstransfer.ui.utils.PreviewAllWindows
 
+// TODO: Use correct URL instead of hard-coded transferLink value
 @Composable
-fun UploadSuccessQrScreen(transferType: TransferType) {
+fun UploadSuccessQrScreen(transferType: TransferType, transferLink: String = "https://chk.me/83azQOl") {
+
+    val context = LocalContext.current
+    val sendIntent = Intent(Intent.ACTION_SEND).apply {
+        putExtra(Intent.EXTRA_TEXT, transferLink)
+        type = "text/plain"
+    }
+    val shareIntent = Intent.createChooser(sendIntent, null)
```

Comment:
Put this logic in its own util file by exposing a `shareString` method or `shareTextPlain` or similar and its "copy to clipboard" alternative as well